CHANGELOG.md

```diff
@@ -77,6 +76,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/).
 - Improve runtime of some unit tests. #1125
 - Run curl OR wget (not both) when attempting to communicate as a new user on
   Linux. #1407
+- Changed trap command signal from INT to TERM. #1406
```

Comment:
You've put this under v1.11.0. This needs to go under "Unreleased".

In addition, put it under "fixed", and just say we fixed the trap post-breach action not producing any output. The specific signal we're using is too low level. Users don't care.